spec/models/mixins/process_tasks_mixin_spec.rb

```diff
@@ -160,10 +160,31 @@ def test_method
           options = {
             :task => "the_task"
           }
-          expect(api_collection).to receive(:the_task).with({})
+          expect(api_collection).to receive(:the_task)
```

Comment:
Why did this change? It feels like this `.with` was the entire purpose of the test as it's name is "uses an empty hash for the args".

Reply:
changed by accident...

Reply:
fixed